civicrm_resources/civicrm_resources.module

```diff
@@ -0,0 +1,132 @@
+<?php
+
+/**
+ * @file
+ * Module to load css and js resources from a directory
+ */
+
+function string_to_slug($string) {
+  $string = str_replace(array('/', ' '), '-', $string);
+  $accents = '/&([A-Za-z]{1,2})(tilde|grave|acute|circ|cedil|uml|lig);/';
+  return urlencode(strtolower(preg_replace($accents, '$1', htmlentities(utf8_decode($string)))));
+}
+
+/**
+ * 
+ */
+function civicrm_resources_add_resources($extention_path, $req_files = array()) {
```

Comment:
First parameter of 'civicrm_resources_add_resources()' is $extension_path while in your examples it's extension's name ('Job Contract (CiviHR)').
Didn't you mean 'civicrm_resources_load()' instead of 'civicrm_resources_add_resources()' in your usage examples?